packages/@sanity/default-layout/src/components/styles/SanityStatus.css

```diff
@@ -48,18 +48,42 @@
   padding: 0.75em;
 }
 
-.notice {
-  /* empty */
+.hasUpdates {
+  display: flex;
+  align-items: center;
+}
+
+.updateIcon {
+  position: relative;
+}
+
+.updateIndicator {
+  position: absolute;
+  top: 0.2em;
+  right: 0.15em;
+  height: 0.5em;
+  width: 0.5em;
+  border-radius: 50%;
+  border: 1px solid var(--main-navigation-color--inverted);
+  box-sizing: border-box;
+}
+
+.updateIndicator[data-severity="notice"] {
+  background: var(--main-navigation-color--inverted);
+}
+
+.updateIndicator[data-severity="low"] {
+  background: #6cb8ff;
```

Comment:
It would be great if we should get rid of this hard-coded color in this update. And use `background: currentColor` instead.

Reply:
What color will current color be? The color of low severity update now is a tone of blue, is there a variable for a blue?

Reply:
I don't think we have a variable for the blue. So we can use `currentColor` which is the same as the text color (https://css-tricks.com/currentcolor/).

Reply:
Yes, but then the low status is no different from the notice status?

Reply:
True, but think that's fine – especially since the "notice" status is only visible when it says "Up to date".